eks/modules/services/aws_auth_cm.tf

```diff
@@ -19,5 +19,5 @@ resource "kubernetes_config_map" "aws_auth" {
     - system:nodes
 ROLES
 }
-depends_on = ["null_resource.force-eks-dependency"]
+depends_on = [null_resource.force-eks-dependency]
 }
```

Comment:
newline needed(last newline comment but assumes will be fixed for rest of the files).

Reply:
> Seems like the changes are not just limited to AWS DNS setup, can we update the description or break down the changes in multiple PRs? Also, can we look at merge conflicts.

Resolved merge conflicts.